docs/reference-docs/app/scaling.md

```diff
@@ -42,6 +43,7 @@ The orchestrator-worker has additional dependencies which can be installed with
 pip install orchestrator-core[celery]
 ```
 
+TODO link to notes on how many instances to run of each
```

Comment:
Do we have existing notes on this? I know SURF has advice here, just not sure what it is and if it's already written down someplace I can link to.

Reply:
Thanks Peter! I replaced the section on autoscaling as you described.